packages/interface/src/features/home/components/Glossary.tsx

```diff
@@ -0,0 +1,66 @@
+import { Heading } from "~/components/ui/Heading";
+
+import { FAQItem } from "./FaqItem";
+
+export const Glossary = (): JSX.Element => (
+  <div className="mt-14 flex flex-col items-center justify-center sm:mt-28 dark:text-white" id="Glossary">
+    <Heading size="6xl">Glossary</Heading>
+
+    <FAQItem
+      description="An individual or team submitting a project or proposal for funding. Applicants aim to attract voter support by showcasing how their project benefits the community or aligns with the funding round's goals."
+      title="Applicant"
+    />
+
+    <FAQItem
+      description="Community members who allocate their voting power (often in the form of credits or tokens) to support projects. Their votes influence both the direct contributions and the matching funds distribution via the quadratic funding mechanism."
+      title="Voters"
+    />
+
+    <FAQItem
+      description="The Coordinator are responsible for deploying the MACI smart contracts, initiating polls, tallying the final results of a vote, and finalizing polls by publishing the final results on-chain."
+      title="Coordinator"
+    />
+
+    <FAQItem
+      description="The team or entity responsible for managing the funding round. Organizers define rules, oversee phases, verify applicant eligibility, and ensure transparency throughout the process."
+      title="Organizers"
+    />
+
+    <FAQItem
+      description="is a community-driven funding method where matching funds are distributed based on the number of contributors and the amounts donated. It amplifies projects with broad support by prioritizing many small contributions over a few large ones, ensuring fair and democratic allocation of resources."
+      title="Quadratic Funding"
+    />
+
+    <FAQItem
+      description="During this phase, applicants submit their proposals, including detailed descriptions, goals, and potential impact. Organizers review and approve submissions to ensure they meet the funding round's criteria."
+      title="Application Phase"
+    />
+
+    <FAQItem
+      description="Voters review the approved proposals and allocate their voting power to the projects they support. Quadratic voting principles apply here, where the cost of additional votes increases quadratically (e.g., 1 vote costs 1 credit, 2 votes cost 4 credits, 3 votes cost 9 credits, etc.)."
```

Comment:
approved proposals? options/candidates/projects?

Reply:
Thinking about adding a "Proposal item" and describing what is it to avoid explaining it on each subsequent item